test/functional/wiki_controller_test.rb

```diff
@@ -25,10 +25,10 @@ def teardown
   end
 
   test 'should get wiki index in alphabetical order' do
-    get :index, params: { order: 'alphabetic' }
+    get :index, params: { sort: 'title' }
```

Comment:
Hi @jywarren , I have a little issue on this, despite passing the sort params...the wikis are still in random order